src/main/java/com/cegeka/academy/service/challengeAnswer/ChallengeAnswerServiceImp.java

```diff
@@ -22,8 +31,13 @@
     private ChallengeAnswerRepository challengeAnswerRepository;
     @Autowired
     private UserChallengeRepository userChallengeRepository;
+    @Autowired
+    private UserChallengeService userChallengeService;
 
-    private Logger logger =  LoggerFactory.getLogger(ChallengeAnswerServiceImp.class);
+    private Logger logger = LoggerFactory.getLogger(ChallengeAnswerServiceImp.class);
+    private static String UPLOAD_ROOT_DIRECTORY = "C:\\photos\\";
```

Comment:
Nu este ok asa. Pe server nu o sa existe aceasta cale. Te rog sa pui in application.properties toate aceste constante si sa citesti pe internet despre @Configuration si @ConfigurationProperties